lib/decimal.ex

```diff
@@ -2,8 +2,8 @@ defmodule Decimal do
   @moduledoc """
   Decimal arithmetic on arbitrary precision floating-point numbers.
 
-  A number is represented by a signed coefficient and exponent such that: `sign
-  * coefficient * 10^exponent`. All numbers are represented and calculated
+  A number is represented by a signed coefficient and exponent such that: ```sign
+  * coefficient * 10^exponent```. All numbers are represented and calculated
```

Comment:
@ericmj I was looking at this and thought it would fix it.
![decimal_v1_1_0_documentation](https://cloud.githubusercontent.com/assets/1635127/11259606/1587e784-8e4a-11e5-985e-2ee8bbb166a5.png)